### ISSUE TITLE
`prisma2 dev`: Asks to save changes into a migration file by running `prisma2 lift save` but gives `Everything up-to-date` in terminal

### ISSUE THREAD

**Issue by deadcoder0904** [prisma/prisma2#651]
Basically looks like 👇

![prisma2 dev](https://user-images.githubusercontent.com/16436270/65889598-26c1bb00-e3bf-11e9-89e5-ae9885e34af4.png)

But when I do it, it gives `Everything up-to-date` in terminal

**Comment by divyenduz** [comment]
I am able to reproduce this, this might be a usability issue and not a bug. Here is what I tried: 

1. Ran `prisma2 dev` of the demo script starter + sqlite3
2. Pressed `d` to see the diff from last save
3. `ctrl + c` to get out of `prisma2 dev`
4. `prisma2 lift save` (even `prisma2 lift up`)
5. Ran `prisma2 dev` again without any changes to the schema file, it still shows this message and the diff message with `d` is still the same - changes since last `prisma2 lift save` is incorrect.
